### ISSUE TITLE
Tabs: required children roles missing

### ISSUE THREAD

**Issue by justintemps** [international-labour-organization/designsystem#767]
- Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children. 
- [role]s are not contained by their required parent element

![Screenshot 2023-12-11 at 16 41 22](https://github.com/international-labour-organization/designsystem/assets/12401179/92021b5c-e593-4d12-a91a-2205b82773e8)
